### ISSUE TITLE
Thermo RAW files will not open after previously importing correctly

### ISSUE THREAD

**Issue by robertsamples** [mzmine/mzmine3#1545]
### Basic information

* My operating system and version: Windows 10/11
* My MZmine version: 3.9.0

### What happened

Last week Thermo RAW files (SIM data) imported without issue, but after acquiring new data with the same method I receive a "no scans found" error with the same MS method when trying to import. I also receive the same error when attempting to open raw files that worked last week and I cannot open saved MZmine projects derived from these data (the projects were not standalone). I am however able to open data I previously exported as mzml. I have attached logs and a zip file containing an example file that will not open now along with a previously converted mzml file that will open. I have tried installing MZmine on a seperate machine and transferring the files there for processing but have the same issue.

[exampledata.zip](https://github.com/mzmine/mzmine3/files/13507164/exampledata.zip)
[mzmine_0_0.log](https://github.com/mzmine/mzmine3/files/13507121/mzmine_0_0.log)
![error](https://github.com/mzmine/mzmine3/assets/64489385/4928369f-0e4a-4682-bc88-a11c016ab764)

**Comment by Annexhc** [comment]
Hi,
I can reproduce your error on macOS as well. Just to clarify, the attached .mzml file is from the exact same .raw file and was created with the .mzml export function from MZmine? What happens when you try to convert the same .raw with MSConvert? Do you also get an error or a valid mzml?

**Comment by robertsamples** [comment]
Yes, the mzml was previously converted with MZmine from this raw file. I obtain the same error with MSConvert. It is very strange behavior as no system or software changes were made since last week.

**Comment by robinschmid** [comment]
internally we use the ThermoRawFileParser. 
There is a pending update internally in MZmine but we had some issues on macOS that we have to fix first. 

Try see if there is a folder mzmine_thermo_raw_parser in Temp and delete it:
C:\Users\###YOUR-USER-NAME###\AppData\Local\Temp\mzmine_thermo_raw_parser\

May also look into your actual MZmine temp folder and also delete it there

**Comment by vaggchem** [comment]
I also have this problem, i could not find the file mentioned to delete it to solve the problem. Any solutions?

**Comment by SteffenHeu** [comment]
please try in mzmine 4 and open a new issue if this persists
